release-schema.json

```
@@ -63,6 +63,16 @@
           "codelist": "methodOfPayment.csv",
           "openCodelist": true,
           "uniqueItems": true
+        },
+        "payee": {
+          "title": "Payee",
+          "description": "The organization that receives the participation fees.",
```

In principle, each object in the list is one fee.

```suggestion
          "description": "The organization that receives the participation fee.",
```